content/ru/function-as-a-service.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Функция-как-сервис (FaaS)
```

Comment:
I have the same question regarding hyphens here :sweat_smile: I'd better remove them.